packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -64,6 +67,14 @@ export function validateTags(tags: any[], description: string) {
     } else {
       validTags.push(tag)
     }
+
+    if (validTags.length >= NEXT_CACHE_TAG_MAX_ITEMS) {
```

Comment:
```suggestion
    if (validTags.length > NEXT_CACHE_TAG_MAX_ITEMS) {
```

The message says "exceeded" so it probably shouldn't happen when equal